src/reducers/status/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './reducers';
```

Comment:
The folder name `status`. What does this refer to? Perhaps change the name to be more descriptive.